carWarGame.py

```diff
@@ -75,10 +82,63 @@ def add_cards(self, cards_pit: list) -> None:
             self.cards.append(card)
 
     def __str__(self) -> str:
-        return f"{self.name.capitalize()} has {self.cards} cards."
+        return f"{self.name.capitalize()} has {len(self)} cards."
+
+    def __len__(self) -> int:
+        return len(self.cards)
+
+
+def greet_and_start() -> bool:
+    """Diplay welcome message to players and ask whether want to start or exit."""
+
+    options = {"1": True, "2": False}
+
+    logger.info("Welcome to CarWar game... Rise your bets!")
+
+    while True:
+        opt = input("Select an option:\n1. Start.\n2. Exit.\n").strip()
+        if opt in options:
+            return options[opt]
+        else:
+            logger.warning("Please enter valid option.")
+
+
+def get_users_name() -> list:
+    """Display message asking for user's name."""
+    players_name = []
+
+    for player in [1, 2]:
+        while True:
+            player_name = input(f"Player{player}, enter name (alphanum only): ")
+            if player_name.isalnum():
+                players_name.append(player_name)
+                break
+            else:
+                logger.warning("Please enter valid name (alphanum only).")
+
+    return players_name
+
+
+def get_num_players() -> list:
+    """Display message asking for the num of players from 1 to 4."""
+    game_players = ["1", "2", "3", "4"]
+    while True:
+        players = input("Please enter number of players (1-4): ").strip()
+        if players.isdigit() and players in game_players:
+            return game_players[: int(players)]
+        else:
+            logger.warning("Please enter valid number of players")
 
 
 if __name__ == "__main__":
-    deck = Deck()
-    player1 = Player("player1", deck.cards[: int(len(deck.cards) / 2)])
-    player1.show_deck()
+    start = greet_and_start()
```

Comment:
Is this PR on work?  please mark as draft in case that it is.

Reply:
Yes, Sr. I am still working on this PR.